swagger/vega/api/v1/core.swagger.json

```diff
@@ -1509,7 +1509,7 @@
         "version": {
           "type": "integer",
           "format": "int64",
-          "description": "A version of the transaction, to be used in the future in case we want to\nimplement changes to the Transaction format."
+          "description": "A version of the transaction, to be used in the future in case changes are implemented \nto the Transaction format."
```

Comment:
```suggestion
          "description": "A version of the transaction, to be used in the future in case changes are implemented into the Transaction format."
```